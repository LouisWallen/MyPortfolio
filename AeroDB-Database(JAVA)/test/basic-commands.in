purge a
push c 6
set a 1 2 3
snapshot
purge a
drop 1
set b 7 8
set b 5 i
set b 5 6
set a t 5
push c 5
push b 1 5.5
push b 1 5
append b t
append b 6
pick b 8
pick b 2
PLUCK b t
pluck b 9
pluck b 1
pop b
pop b
pop b
pop b
pop b
drop r
rollback 4
rollback r
checkout 2
checkout 6.6
del b
del c
set a 6 3 4 1 5
min a
max a
SUM a
len a
rev a
get a
append a 6
uniq a
set c 7 8
del a
set h 8 8
del c
set a 6 7 8 5 4
sort a
