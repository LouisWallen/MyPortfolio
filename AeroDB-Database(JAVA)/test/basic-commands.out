> ok

> no such key

> ok

> saved as snapshot 1

> ok

> ok

> ok

> Please input integers in value of the key

> ok

> Please input integers in value of the key

> no such key

> Please input integers in value of the key

> ok

> Please input integers in value of the key

> ok

> index out of range

> 1

> Please input integers in index of the key

> index out of range

> 5

> 1

> 5

> 6

> 6

> nil

> Please input integers in id of the snapshot

> no such snapshot

> Please input integers in id of the snapshot

> no such snapshot

> Please input integers in id of the snapshot

> ok

> no such key

> ok

> 1

> 6

> 19

> 5

> ok

> [5 1 4 3 6]

> ok

> ok

> ok

> ok

> ok

> ok

> ok

> ok

> No input. Bye
bye
